lib/modules/apostrophe-pieces-submit-widgets/index.js

```diff
@@ -28,20 +31,44 @@ module.exports = {
     };
 
     self.beforeInsert = function(req, piece, callback) {
+      var person = {
+        toName: "admin",
+        email: process.env.SENDER_EMAIL_ADDRESS,
+        subject: piece.subject,
+        details: piece.details,
+        clientEmail: piece.email,
+        clientName: piece.title
+      };
+      var detailsForSendMany = {
+        templateName: "template",
+        context: person,
+        subject: person.subject,
+        toAddresses: [person.email],
+        senderEmailAddress: person.senderEmailAddress,
+        replyToAddress: person.replyToAddress,
+        ccAddresses: null,
+        bccAddresses: null
+      };
+      sendMany(detailsForSendMany, function(error, result) {
```

Comment:
Could we use just [`sendemail`](https://github.com/dwyl/sendemail/blob/master/lib/index.js#L137) here instead. It looks as though it would suffice and be simpler

Reply:
It would seem that way, however, we will need to use sendMany as we will eventually be sending to multiple email addresses.

Reply:
Makes sense 👍 It sure would be nice if `sendemail` could have the option to take an array of emails. (I've made the suggestion [here](https://github.com/dwyl/sendemail/issues/86#issuecomment-373430000), let's see what @nelsonic thinks)